src/components/chips/js/chipsController.js

```diff
@@ -584,16 +582,21 @@ MdChipsCtrl.prototype.validateModel = function() {
   this.ngModelCtrl.$validate(); // rerun any registered validators
 };
 
+MdChipsCtrl.prototype.updateNgModel = function() {
+  this.ngModelCtrl.$setViewValue(this.items.slice());
```

Comment:
This line seems to be related to https://github.com/angular/material/issues/11304 and https://github.com/angular/material/issues/11301.